packages/backend/src/projectAdapters/dbtGitProjectAdapter.ts

```diff
@@ -24,6 +28,26 @@ export type DbtGitProjectAdapterArgs = {
     cachedWarehouse: CachedWarehouse;
 };
 
+const stripTokensFromUrls = (raw: string) => {
+    const pattern = /\/\/(.*)@/g;
+    return raw.replace(pattern, '//*****@');
+};
+
+const gitErrorHandler = (e: Error) => {
+    if (e.message.includes('Authentication failed')) {
+        throw new AuthorizationError(
+            'Git credentials not recognized for this repository',
+            { message: e.message },
```

Comment:
are you sure the Authentication failed error does not include emails ? Should we do `{ message: stripTokensFromUrls(e.message));}` just in case ?